### ISSUE TITLE
Accessibility Audit: Patients page flows

### ISSUE THREAD

**Issue by mehansen** [CDCgov/prime-simplereport#5780]
### Background
The SimpleReport team performs quarterly accessibility audits as a proactive approach to ensure our products comply with Section 508 of the Rehabilitation Act.

### Action requested
Use Axe DevTools to perform an accessibility audit of the patient management user flows. Do the page scan as well as any relevant guided tests.
Pages to visit:
* `/app/patient`
* `/app/patients`
* `/app/add-patient`

Use cases to test:
* view patients list with no, 1, and multiple patients
* use patient search with match(es) and no match
* archive patient modal
* add a patient
* edit a patient

### Acceptance Criteria
Full audit completed and tickets created for any accessibility violations.

**Comment by BobanL** [comment]
Tickets created:
* https://github.com/CDCgov/prime-simplereport/issues/5817
* https://github.com/CDCgov/prime-simplereport/issues/5821
